modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetTrafficSourcesMVCResourceCommand.java

```diff
@@ -135,6 +140,96 @@ protected void doServeResource(
 		}
 	}
 
+	private SocialMediaTrafficDetails _getSocialMediaTrafficDetails() {
+		return new SocialMediaTrafficDetails(
+			Arrays.asList(
```

Comment:
if you want to add this mock, that is totally fine, please, use a feature flag to activate it :)